overlord/managers_test.go

```diff
@@ -7482,6 +7482,11 @@ func (s *mgrsSuiteCore) testRemodelUC20WithRecoverySystem(c *C, encrypted bool)
 	})
 	defer restore()
 
+	restore = fdestate.MockSecbootGetPCRHandle(func(devicePath, keySlot, keyFile string) (uint32, error) {
+		return 42, nil
```

Comment:
FWIW, I would return an int which looks like the actual handles we expect, such that when looking at the logs you'd see familiar values, so eg. 0x1880002